views/site/team.php

```diff
@@ -34,7 +34,7 @@
                         <?php foreach($row as $i=>$member):?>
                             <div class="col-sm-4 col-md-2 person-card <?= ($i==0 && count($row)<6)?'col-md-offset-'.(6-count($row)):'' ?>">
                                     <div class="avatar">
-                                        <img src="<?= Html::encode(Yii::getAlias($member['photo'] ?? '@web/image/team/noimage.png')) ?>" class="img-responsive" alt="" />
+                                        <img src="<?= Html::encode(Yii::getAlias($member['photo'] ?? '@web/image/team/noimage.png')) ?>" class="img-responsive" alt="<?= Html::encode($member['name'])?>" />
```

Comment:
As far as I remember, `alt` is read by screen readers so it will repeat name twice and that's not good.

Reply:
For example, there is also a description in alt
https://www.yiiframework.com/books

Reply:
https://www.w3.org/WAI/tutorials/images/decision-tree/

Reply:
i update PR to fix this issue https://github.com/yiisoft-contrib/yiiframework.com/issues/599